Ginger/Ginger/Run/RunSetsExecutionsHistoryPage.xaml.cs

```diff
@@ -672,20 +710,36 @@ private BusinessFlow GetBusinessFlowByName(string name)
             return bf;
         }
 
-        private GingerCore.Activity? GetBusinessFlowActivityByName(BusinessFlow bf, string activityName)
+        private bool ActivityExistInSharedRepository(string name)
```

Comment:
Change this method to return the object instead of flag to avoid multiple methods

Reply:
removed the unnecessary method.